docx/oxml/bookmark.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Custom element classes for bookmarks
+"""
+
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from . import parse_xml
+from .ns import nsdecls, qn
+
+from .xmlchemy import (BaseOxmlElement, OneOrMore,
+    RequiredAttribute, ZeroOrOne)
+
+from .simpletypes import ST_RelationshipId, ST_String
+from docx.oxml import OxmlElement
+
```

Comment:
* Imports appear in alphabetical order (with some exceptions that don't apply here, we'll see those later).
* In the past, I used relative imports whenever possible (e.g. `.xmlchemy`). I've since learned better, and all new work should be explicit (e.g. `from pptx.oxml.xmlchemy ...`). Also, as we touch the imports in a file, this is something to clean up. I haven't seen any need to do it globally though, which is why you'll still encounter it from time to time.